openapi.yaml

```diff
@@ -10307,6 +10307,94 @@ paths:
             --region us-east \
             --assignments.address 2001:db8:3c4d:15::/64 \
             --assignments.linode_id 123
+  /networking/ips/share:
+    x-linode-cli-command: networking
+    post:
+      servers:
+      - url: https://api.linode.com/v4beta
```

Comment:
this isn't in beta

Reply:
No problem reverting, but I'm getting {"errors": [{"reason": "Method Not Allowed"}]} when trying to hit the v4 server

Reply:
added v4 server in the next line and removed beta status references -- beta tag still shows up in the docs at the moment, but the CLI won't work with the command until the v4 server issue is fixed for this command otherwise